pkg/util/interrupt/child_linux.go

```diff
@@ -24,4 +24,4 @@ import (
 )
 
 // childSignals are the allowed signals that can be sent to children in Unix variant OS's
```

Comment:
@bparees you may want to update this as well as per your changes in the source-to-image pr, so we keep it all consistent?